src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const allEmployers = [...document.querySelectorAll('li')];
+
+const sortBySalaries = allEmployers.sort((a, b) => {
+  const salaryA = a.getAttribute('data-salary').slice(1).split(',').join('');
```

Comment:
`getAttribute('data-salary').slice(1).split(',').join('');` this code is duplicated, so make sense to move it to the function